src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_account_commands.py

```diff
@@ -38,7 +39,7 @@ def test_create_delete_account(self):
         ])
 
         # delete and recheck
-        self.cmd("az netappfiles account delete --resource-group {rg} --account-name '{acc_name}'")
+        self.cmd("az netappfiles account delete --yes --resource-group {rg} --account-name '{acc_name}'")
```

Comment:
It's a breaking change for the delete command. You may need to remove the confirmation in delete commands currently. You can add that confirmation in the next breaking change window.